File: RepainterAPI/DataEntities/Package/Package.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataEntities.Package
{
    public class Package
    {
        public int PackageId { get;set; }
        public bool PackageTabId { get;set; }
        public bool GeneratedId { get;set; }
        public string PackageType { get;set; }
        public string ProductCode { get;set; }
        public string Type { get;set; }
        public string SelectedCode { get; set; }
        public decimal Amount {  get;set; } 
        public string Specification { get;set; }
        public string Condition { get;set; }
        public string Remarks { get;set; }
        public decimal SectionTotalPreTax { get;set; }
        public decimal SectionTotalPostTax { get;set; }
        public int CustomerId { get;set; }
        public bool Deleted { get;set; }
        public DateTime CreatedOn { get;set; }  
        public int CreatedBy { get;set; }
        public int LastModifiedBy { get; set; }
        public DateTime LastModifiedOn { get; set; }
    }
}

File: Repainters/src/app/Shared/models/package.ts
export class Package
{
    packageId?:number;
    packageTabId!:boolean;
    generatedId!:boolean;
    customerId!:number;
    packageType!:string;
    productCode!:string;
    selectedCode!:string;
    type!:string;
    amount!:number;
    specification!:string;
    condition!:string;
    remarks!:string;
    sectionTotalPreTax!:number;
    sectionTotalPostTax!:number
    deleted!:boolean;
    createdBy?: number;
    lastModifiedDate?: Date;
    lastModifiedBy?: number;
    createdOn?: Date;
    constructor() {
        this.createdOn = new Date();
        this.lastModifiedDate = new Date();
    }
}